sourcecode/apis/contentauthor/app/Libraries/H5P/LtiToH5PLanguage.php

```diff
@@ -8,21 +9,24 @@
 
 namespace App\Libraries\H5P;
 
-
 class LtiToH5PLanguage
 {
-    public static function convert($ltiLanguage = 'en-gb', $defaultLanguage = 'en-gb')
+    public static function convert($ltiLanguage = 'en-gb', $defaultLanguage = 'en-gb'): string
```

Comment:
Whole function could be simplified as:

~~~php
$language = $ltiLanguage ?: $defaultLanguage ?: 'en';

return explode('-', $language)[0];
~~~

An exception should probably be thrown if `$language` ends up being a single-character string.